src/index.html

```diff
@@ -4,14 +4,148 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="inputs-block">
+        <legend>Personal information</legend>
+        <label class="input-line">
+          Surname:
+          <input
+          name="surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            name="surname"
            autocomplete="off"
          >
```
fix spaces, check code style in other places by [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

Reply:
understood, thanks)